src/cpu/CPUCore.cc

```diff
@@ -518,6 +518,29 @@ static constexpr void toHex(byte x, std::span<char, 3> buf)
 	buf[1] = toHex(x & 15);
 }
 
+template<typename T> void CPUCore<T>::disasmBinCommand(
+	Interpreter& interp, std::span<const TclObject> tokens, TclObject& result) const
+{
+	if (tokens.size() < 3) {
+		throw CommandException("wrong # args: should be \"debug disasm_bin value ?addr?\"");
```

Comment:
There is no easy way to use it now, since I need to create a class that inherits from `Command` to use `checkNumArgs()` and since `disasmBin` should not be connected to `CPUCore` anyway, it needs to live inside its own command (something like a `DasmCommand` class).